### PR TITLE
add rust-toolchain.toml file to auto select right rustc version

### PR DESCRIPTION
**Pull request by Pana**

### CODE REVIEW

File: crates/rpc/rpc-builder/src/lib.rs
Comment (mattsse) [inline crates/rpc/rpc-builder/src/lib.rs:1]: changes unrelated?

File: rust-toolchain.toml
Comment (mattsse) [inline rust-toolchain.toml:1]: do we want this @Rjected @DaniPopes ?

File: crates/rpc/rpc-builder/src/lib.rs
Comment (Pana) [reply crates/rpc/rpc-builder/src/lib.rs:1]: It's a small code optimization

**Comment by DaniPopes** [comment]
I personally don't use this, I use nightly by default with nightly rustflags; this forces stable, which makes me usually just remove the file so it actually even compiles

**Comment by mattsse** [comment]
yeah same, don't think we won't this then

**Pull request closed without merging**